add-on/src/lib/dnslink.js

```diff
@@ -1,25 +1,26 @@
 'use strict'
 /* eslint-env browser */
 
-const debug = require('debug')
+import Pqueue from 'p-queue'
+
+import debug from 'debug'
+import IsIpfs from 'is-ipfs'
+import LRU from 'lru-cache'
+import { offlinePeerCount } from './state.js'
+import { ipfsContentPath, sameGateway, pathAtHttpGateway } from './ipfs-path.js'
```

Comment:
nit: spacing - I personally prefer organizing imports in four (potentially 5 groups depending on how large your 1st party import sections are): 
1. stdlib imports; 
2. 3p imports; 
3. 2p imports (internal to company but not to team)
4. 1p imports (internal to team)
5. local/relative imports (internal to project)

eslint-plugin-imports allows you to automate this